data-processing/requirements.txt

```diff
@@ -9,6 +9,9 @@ pysbd
 reportlab
 requests
 TexSoup==0.2.0
+nltk==3.4.5
```

Comment:
Is this still necessary? I couldn't find references to it elsewhere in the code.